cirq/ops/measurement_gate.py

```diff
@@ -81,7 +88,8 @@ def with_bits_flipped(self, *bit_positions: int) -> 'MeasurementGate':
             new_mask[b] = not new_mask[b]
         return MeasurementGate(self.num_qubits(),
                                key=self.key,
-                               invert_mask=tuple(new_mask))
+                               invert_mask=tuple(new_mask),
+                               qid_shape=self._qid_shape)
```

Comment:
This is fine here - but it popped a question in my head which might result in a new issue: how do we (do we?) apply an "invert_mask" on qudits?

Reply:
I don't think we do that anywhere. Frankly I'm not sure that invert mask as a thing is particularly useful, but we added it early on (well before we had any notion of qudits in cirq).

Reply:
Got it! 

Re qudits: I found https://github.com/quantumlib/Cirq/pull/2079 explaining that it flips |0> with |1> and leaves |2+> alone. 
Re general usefulness: It seems (at least based on https://github.com/quantumlib/Cirq/issues/1943) that some users use invert mask